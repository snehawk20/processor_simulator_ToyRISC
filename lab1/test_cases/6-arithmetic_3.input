	.data
a:
	100
d:
	-5
n:
	5
