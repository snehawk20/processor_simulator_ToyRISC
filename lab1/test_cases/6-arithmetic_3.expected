65531		: 80
65532		: 85
65533		: 90
65534		: 95
65535		: 100
